conjur/cli_actions.py

```diff
@@ -202,6 +203,26 @@ def handle_variable_logic(args: list = None, client=None):
         variable_controller.set_variable()
 
 
+def handle_role_logic(args: list = None, client=None):
+    """
+    Method wraps the role call logic
+    """
+    role_logic = RoleLogic(client)
+    role_controller = RoleController(role_logic=role_logic)
+    role = Role.from_full_id(args.identifier)
```

Comment:
It looks like for other commands, we just pass in the arguments as is from the CLI and leave the controller to create an object from the ID. See for example [list_logic.py](https://github.com/cyberark/cyberark-conjur-cli/blob/a2f9f912aa9696b1bda81122745647c43fa9ac49/conjur/logic/list_logic.py#L37)